src/components/ButtonMenu.tsx

```diff
@@ -47,6 +47,8 @@ type MenuItemBase = {
   onClick: string | VoidFunction;
   disabled?: boolean;
   destructive?: boolean;
+  /** If using a string as the onClick value, and setting this to true will force the browser to open the link in the same browser window as a new tab */
+  forceSameBrowserTab?: boolean;
```

Comment:
Ah okay, I think I misunderstood. Fwiw `forceSameBrowserTab` made me think we wanted literally the same tab, i.e. `_self`, behavior.

But we're not really concerned with `_self` vs. `_blank`, we always want `_blank`, and we always want it to be "new tab _in the same window".

And the issue was that Safari seeing `noopener.noreferrer` meant it was "new tab but _new window_"? 

If my new understanding is correct, it seems like the "fixed for Safari" behavior is probably want everyone wants, and we should add flag to request the behavior.

I think personally I'd be fine with something like changing line 37 to be:

```
// We want to do `window.open(url, "_blank", "noopener,noreferrer")` but that Safari treats
// that as "open in new window", see ...url...
//
// So we do this instead, and at least null out the opener
// as a way to manually mimic the `"noopener"` flag.
window.open(onClick, "_blank") as Window).opener = null;
```

Reply:
@stephenh changes updated ✅